### ISSUE TITLE
Search bar doesn't render properly

### ISSUE THREAD

**Issue by sadan4** [zt64/tau#8]
![image](https://github.com/zt64/tau/assets/117494111/7d1e11c1-c8bb-446b-a730-25808813e978)
![image](https://github.com/zt64/tau/assets/117494111/b2293366-5548-4858-a1e0-8787bb1e119e)

**Comment by keli5** [comment]
Confirmed, issue appears on Windows as well

**Comment by keli5** [comment]
man i suck at using github

**Comment by keli5** [comment]
this seems to be fixed on windows, perhaps due to a compose update or something, @zt64 fixed on linux?
